.ado/templates/e2e-testing-windows.yml

```diff
@@ -0,0 +1,71 @@
+parameters:
+  buildSuccess: # Whether or not the initial build steps succeeded
+
+steps:
+  - task: DownloadSecureFile@1
+    name: UwpCertificate
+    inputs:
+      secureFile: FluentTester_TemporaryKey.pfx
+    displayName: "Download UWP App Certificate"
+
+  - powershell: |
+      certutil –f –p password –importpfx $(UwpCertificate.secureFilePath)
+    displayName: "Add Certificate to Personal Store"
+  - task: DownloadSecureFile@1
+    name: AzureCredentials
+    inputs:
+      secureFile: "installcredprovider.ps1"
+    displayName: "Download Azure Credentials Plugin for NuGet"
+
+  - task: PowerShell@2
+    inputs:
+      filePath: $(AzureCredentials.secureFilePath)
+      arguments: "-AddNetfx -Force"
+    displayName: "Install Azure Credentials Plugin for NuGet"
+
+  - script: |
+      yarn windows
+    workingDirectory: apps\windows
+    displayName: "run UWP app"
+  # Wait for app to launch. A workaround to avoid WinAppDriver error: Failed to locate opened application window with appId
+  - powershell: |
+      Start-Sleep -Seconds 90
+    displayName: "Wait for app to launch"
+  # Kill FluentTester, leave server up and running. This was the only way I could get the server continuously running
+  # as a separate process.
+  - powershell: |
+      Stop-Process -Name FluentTester
+    displayName: "Kill FluentTester Process"
+  - script: |
+      yarn e2etest
+    workingDirectory: apps\windows
+    condition: succeeded()
+    displayName: "run UWP E2E tests"
+  # The following conditions (using buildSuccess variable) make it so these tasks run even if the E2E tasks fails,
+  # but not if the initial repo build steps fail.
+  - template: e2e-publish-artifacts.yml
+    parameters:
+      applicationType: UWP
+      platform: windows
+      buildArtifacts: ${{parameters.buildSuccess}}
+
+  # Necessary for Appium server to run twice (for UWP and Win32). One instance at a time
+  - powershell: |
+      Stop-Process -Name node
+    displayName: "Kill node Process"
+    condition: and (succeededOrFailed(), eq(${{parameters.buildSuccess}}, 'success'))
+  # Performance Reasons
+  - powershell: |
+      Start-Sleep -Seconds 30
+    displayName: "Performance Delay"
+    condition: and (succeededOrFailed(), eq(${{parameters.buildSuccess}}, 'success'))
+  - script: |
```

Comment:
Nbd, but I might move the win32 e2e to the top. Less setup involved - you can fail faster if there's something fundamentally broken

Reply:
The way I have it right now, both E2E tests runs (UWP+Win32) will run even if the other fails. I did this because there could be failures in one that don't affect the other. The order doesn't r matter functionality wise, but I agree having Win32 first is more logical. I will change it in my newest commit.

Reply:
Updated in newest commit